addons/popochiu/engine/popochiu.gd

```diff
@@ -1023,7 +1024,8 @@ func _on_character_spoke(chr: PopochiuCharacter, msg := '') -> void:
 
 
 func _on_graphic_interface_unblocked() -> void:
-	clicked = null
+	pass
```

Comment:
Duh?!

Can you please explain?

Reply:
My fault. I forgot to remove the function.